client/state/selectors/is-fetching-viewers.js

```diff
@@ -0,0 +1,3 @@
+export default function isFetchingViewers( state, siteId ) {
```

Comment:
Yes, that's correct, and the same for action creators. That helps to guarantee that no matter what selectors or action creators are being made use of, the state will get initialised.

And since each module only executes once, it doesn't matter that we're importing this in a whole bunch of places; only the first one will run.

Reply:
Done via https://github.com/Automattic/wp-calypso/pull/48137/commits/9b824856fe14e5befa028fbde59785ac6ce02008